pkgs/development/libraries/tbb/2021_8.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchurl
+, cmake
+}:
+
+let
+  common = import ./common.nix { inherit lib; };
+in
+stdenv.mkDerivation rec {
+  pname = "tbb";
+  version = "2021.8.0";
+
+  outputs = [ "out" "dev" ];
+
+  src = fetchFromGitHub {
+    owner = "oneapi-src";
+    repo = "oneTBB";
+    rev = "v${version}";
+    hash = "sha256-7MjUdPB1GsPt7ZkYj7DCisq20X8psljsVCjDpCSTYT4=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  patches = [
+    # Fix musl build; vendored from https://github.com/oneapi-src/oneTBB/pull/899
+    ./musl.patch
```

Comment:
I suggest using `fetchpatch` here against https://github.com/oneapi-src/oneTBB/commit/493774eb57f9c424fc8907d137665e687861ad94.patch URL. It should be more clear it's not a `nixpkgs`-specific patch.

Reply:
Maybe the reason it was made as a file is that it's not yet merged and if so, the link will break. But we can update it then.